src/components/ProductCard.tsx

```diff
@@ -7,7 +7,7 @@ import ProductCardShowMoreButton from "./productCardShowMoreButton";
 export type Content = {
   colours: string[];
   image: string[];
-  showMoreImage: string[];
+  backOfCardImage: string[];
```

Comment:
This still doesn't quite explain why an Image is an array of strings, I would think that there might be a reason why you can have multiple images on the back of a card

Reply:
no particular reason. Just a pattern I'm used to using. Changed to single string.

Reply:
it's due to having multiple wallets shown per card. Thus multiple back side images are shown per object as well.

Reply:
I don't quite understand what this means, can you give me an example or create a loom of what you're referring to? A card has two sides, front or back. How do you represent multiple wallets per card?

Reply:
the structure of the product card is set where each card is an object with arrays storing all the images. Each position in the array is in order with the colours. Thus obj.colours[0] is attached to obj.images[0]. Therefore, obj.backOfCardImage[0] would also be attached to the particular product in  a card. Perhaps, a more accurate statement is that each card represents a single card with differing colours leading to different sets of images.

Reply:
Got it, if that's your intention then I would suggest you reevaluate how you name these properties because that's not clear at all. image is singular, so seeing an array there is odd, backOfCardImage is singluar but also see an array. 

Another thing to ask yourself is, what are some of the trade offs of using a data structure like this? What kind of edge cases have you not considered. How can you make this better and less error prone for yourself and in the future?

Reply:
I would need to have index's to access of the values. As well, if the position data order was to be rearranged it would break the application. To avoid this I would have to structure the data in such a way where each set would have a unique object attached to each product card colour some like {colour: "string", frontImage:  "string", backImage: "string" .... etc}.